### DISCUSSION TITLE
How to return to the Ensemble app from the Webview

### DISCUSSION THREAD

**Discussion by gaurikalp** [EnsembleUI/ensemble#166] [General]
How to return to the Ensemble app from the Webview.
We want the page to redirect to Ensemble app rather than another webpage after submission from Webview.

**Comment by kmahmood74** [comment]
created this - https://github.com/EnsembleUI/ensemble/issues/176

when someone goes from one url to another in the webview, you will get an event. Then handler of that event can then close the dialog that has the webview in it

**Reply by kmahmood74** [reply]
it's being reviewed, should be in by Tuesday

**Comment by kmahmood74** [comment]
hi @gaurikalp this issue has been fixed. Please get the latest Ensemble code and try the WebView from the kitchen sink on your iPhone or Android simulator (or Phone) 

Note that the events won't work on the web. They only work inside the WebView on iOS or Android

**Reply by gaurikalp** [reply]
Yes, it has been implemented in the Aplex app, however there is currently just one style-related issue: when we set expanded: true, the WebView is hidden. However, if we set a specific height, the WebView is displayed.

**Reply by kmahmood74** [reply]
hi @gaurikalp could you please send us the link to the screen that has that issue and also tell us where.
